Song.Tests/SongWriterTests.cs

```diff
@@ -0,0 +1,91 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Runtime.ConstrainedExecution;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Song.Tests
+{
+    public class SongWriterTests
```

Comment:
Your tests aren't unit tests, because you're testing both SongWriter and VerseWriter.

Reply:
Looking into mocking using Moq: https://github.com/devlooped/moq